homeassistant/components/squeezebox/media_player.py

```diff
@@ -244,13 +246,16 @@ def rediscovered(self, unique_id: str, connected: bool) -> None:
                 self._remove_dispatcher()
 
     @property
-    def state(self) -> MediaPlayerState | None:
+    def state(self) -> MediaPlayerState:
         """Return the state of the device."""
         if not self._player.power:
             return MediaPlayerState.OFF
-        if self._player.mode:
-            return SQUEEZEBOX_MODE.get(self._player.mode)
-        return None
+        if self._player.mode and self._player.mode in SQUEEZEBOX_MODE:
+            return SQUEEZEBOX_MODE[self._player.mode]
+        _LOGGER.error(
+            "Received unknown mode %s from player %s", self._player.mode, self.name
+        )
+        return MediaPlayerState.IDLE
```

Comment:
`None` would render the entity as unknown, that would be more correct here right?

Reply:
Yes, good point.